compiler/rustc_resolve/src/lib.rs

```diff
@@ -1686,21 +1686,43 @@ impl<'a> Resolver<'a> {
             .or_insert_with(|| self.arenas.alloc_name_resolution())
     }
 
+    // Test if AmbiguityError ambi is any identical to any one inside ambiguity_errors
+    fn is_identical_error(&mut self, ambi: &mut AmbiguityError<'_>) -> bool {
```

Comment:
Why does this take `&mut`?

Reply:
Removed `mut` because it won't change it. That was a mistake of beginner like me. Thank you for the suggestion.